### ISSUE TITLE
[suggestion] Allow child cubes to have a different scale from their parent groups

### ISSUE THREAD

**Issue by cybercat5555** [iChun/Tabula#91]
Exactly like what it sounds like. If the parent cube is say, 1-1-1 all around, a child cube could be .9-.9-.9 all around, and still act as a proper child cube. Currently the only way to create an effect similar to this and still having the parts grouped is by placing 4 separate cubes very close together, which is just a hassle at times

**Comment by iChun** [comment]
There's no hook for adjusting the scale of a child cube because when
calling the render of the parent cube, the parent cube calls the render for
the child cube in its render code.

**Comment by cybercat5555** [comment]
Ahh, I see, that makes sense. Thanks for the response nonetheless

**Comment by Poulpynou** [comment]
I'm reviving this thread because it does work in the minecraft 1.12 (I havn't check in the previous versions).

I'm using this in my armor model at it works perfectly :
```
         this.chestSub = new ModelRenderer(this, 0, 0);
         this.chestSub.setRotationPoint(0.0F, 0.0F, 0.0F);
         this.chestSub.addBox(-4.0F, 0.0F, -2.0F, 8, 12, 4, **0.3F**);
```
The scale is indeed different from the parent box chestSub. Could you unlock this feature in Tabula? Currently, I have to create the whole model without parent to decide the scale, export, and add parent while manually substracting the coordinates... It's not that fun :(

**Comment by gegy1000** [comment]
Addressed in #103

**Comment by iChun** [comment]
As iChunUtil and Tabula has been redone-over (again) for 1.15+. I am closing issues pertaining to 1.12.2 and earlier. 

My apologies that I've not been able to address your issue.
